app/src/main/java/com/example/parkingsystem/mvp/view/ParkingSpaceReservationView.java

```diff
@@ -0,0 +1,60 @@
+package com.example.parkingsystem.mvp.view;
+
+import android.app.Activity;
+import android.app.DatePickerDialog;
+import android.app.TimePickerDialog;
+import android.content.Context;
+import com.example.parkingsystem.databinding.ActivityReservationSpaceParkingBinding;
+import com.example.parkingsystem.mvp.contracts.ParkingSpaceReservationContract;
+import com.example.parkingsystem.mvp.view.base.ActivityView;
+import java.util.Calendar;
+
+public class ParkingSpaceReservationView extends ActivityView implements ParkingSpaceReservationContract.ParkingSpaceReservationView {
+    private ActivityReservationSpaceParkingBinding binding;
+    private final String TEXT_AT = " at ";
+
+    public ParkingSpaceReservationView(Activity activity, ActivityReservationSpaceParkingBinding binding) {
+        super(activity);
+        this.binding = binding;
+    }
+
+    @Override
+    public void showDatePickerDialog(DatePickerDialog.OnDateSetListener listener) {
+        Calendar calendar = Calendar.getInstance();
+        int day = calendar.get(Calendar.DAY_OF_MONTH);
+        int month = calendar.get(Calendar.MONTH);
+        int year = calendar.get(Calendar.YEAR);
+        Context context = getContext();
+        if (context != null) {
+            DatePickerDialog datePicker = new DatePickerDialog(context, listener, year, month, day);
+            datePicker.show();
+        }
+    }
+
+    @Override
+    public void showDateAndTimeStart(String date, String time) {
+        binding.textParkingSpaceReservationPickerBegin.setText(date + TEXT_AT + time);
```

Comment:
please use string resources not TEXT_AT like

"%1$s at %2$s"

and then getString(<string_name>, date, time)

Reply:
Changed!